gatsbyjs/gatsby
# fix(docs): Correct typo in eCommerce tutorial where Gatsby is misspelled

## Description

This change is to fix a typo in Ecommerce docs - 'Gastby' instead of 'Gatsby'.

## QA

Review the code change, verify the spelling is correct and no other changes have been introduced.
